zio-cli/shared/src/main/scala/zio/cli/CliApp.scala

```diff
@@ -56,6 +57,75 @@ object CliApp {
     def printDocs(helpDoc: HelpDoc): UIO[Unit] =
       printLine(helpDoc.toPlaintext(80)).!
 
+    def checkAndGetOptionsFilePaths(topLevelCommand: String): List[String] = {
```

Comment:
This is side-effecting, so should return `Task[List[String]]` for consistency with the rest of ZIO ecosystem.